WalletWasabi.Bench/TransactionProcessorBench.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using NBitcoin;
+using WalletWasabi.KeyManagement;
+using WalletWasabi.Models;
+using WalletWasabi.Services;
+using System.IO;
+using WalletWasabi.Helpers;
+using System.Runtime.CompilerServices;
+using NBitcoin.Protocol;
+using WalletWasabi.Stores;
+using System.Net;
+
+namespace WalletWasabi.Bench
+{
+	[ClrJob(baseline: true), CoreJob]
+	[RPlotExporter, RankColumn]
```

Comment:
This is new for me pls add more comments.

Reply:
`RankColumn` sort the results from best to worst time. RPlotExplorer is to request the genearion of the `.R` file (I have remove it). Thank you.